app/services/idv/steps/verify_wait_step_show.rb

```diff
@@ -68,6 +69,19 @@ def async_state
       def delete_async
         flow_session.delete(verify_step_document_capture_session_uuid_key)
       end
+
+      def idv_result_to_form_response(result:, state: nil, state_id_jurisdiction: nil, extra: {})
+        state_id = result.dig(:context, :stages, :state_id)
+        state_id[:state] = state if state && state_id
+        if state_id_jurisdiction && state_id
+          state_id[:state_id_jurisdiction] = state_id_jurisdiction
+        end
```

Comment:
unsure if this helps readability (or fits w/in line length), just an idea:
```suggestion
        if state_id
          state_id[:state] = state if state
          state_id[:state_id_jurisdiction] = state_id_jurisdiction if state_id_jurisdiction
        end
```